obsplus/events/get_events.py

```diff
@@ -145,6 +146,22 @@ def _get_ids(df, kwargs) -> set:
     return set(df.event_id[:limit])
 
 
+def _handle_dateline_transversal(filt, df, kwargs):
+    """Check if dateline should be transversed by query."""
+    # if longitudes aren't being used bail out
+    if not {"minlongitude", "maxlongitude"}.issubset(set(kwargs)):
+        return filt, kwargs
+    # if dateline is not to be transversed by query bail out
+    minlong, maxlong = kwargs["minlongitude"], kwargs["maxlongitude"]
```

Comment:
```suggestion
    minlong, maxlong = kwargs["minlongitude"], kwargs["maxlongitude"]
    minlong = ((minlong + 180) % 360) - 180
    maxlong = ((maxlong + 180) % 360) - 180
```
this should help cope with people (like me) doing dumb things?